corehq/apps/app_manager/static/app_manager/template_apps/wash_before_cond_case_update/multimedia.json

```diff
@@ -0,0 +1,16 @@
+{
+  "commcare/image/module0_form0_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module0_form0_en.png",
+  "commcare/image/module1_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module1_en.png",
+  "commcare/image/module2_form1_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module2_form1_en.png",
+  "commcare/image/module2_form0_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module2_form0_en.png",
+  "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module2_form0_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module1_form0_en.png",
+  "commcare/image/module3_form0_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module3_form0_en.png",
+  "commcare/image/module1_form1_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/wash/module1_form1_en.png"
+}
```

Comment:
This seems like an accidental commit, please rebase this commit to remove the file.

Reply:
I thought it was just part of the app so harmless, but clearly unnecessary so I've removed.